webpack/ForemanTasks/Components/TasksTable/formatters/selectionHeaderCellFormatter.js

```diff
@@ -5,6 +5,7 @@ export default (selectionController, label) => (
   <TableSelectionHeaderCell
     label={label}
     checked={selectionController.allPageSelected()}
+    disabled
```

Comment:
this looks like it will be always disabled, shouldn't it depend on the perms?

Reply:
"I'll just try hardcoing it to see if it works and the change it"
*fast forward five minutes*
"Ok, it works, done"

Reply:
been there :+1: